lago/vm.py

```diff
@@ -226,13 +226,22 @@ def _libvirt_xml(self):
         with open(_path_to_xml('dom_template.xml')) as xml_fd:
             dom_raw_xml = xml_fd.read()
 
-        qemu_kvm_path = [
-            path
-            for path in [
-                '/usr/libexec/qemu-kvm',
-                '/usr/bin/qemu-kvm',
-            ] if os.path.exists(path)
-        ].pop()
+        capabilities_raw_xml = self.libvirt_con.getCapabilities()
+        capabilities_xml = lxml.etree.fromstring(capabilities_raw_xml)
+        qemu_kvm_path = capabilities_xml.findtext(
+            "guest[os_type='hvm']/arch[@name='x86_64']/domain[@type='kvm']"
+            "/emulator"
+        )
+
+        if not qemu_kvm_path:
+            # Nested virtualization not available
```

Comment:
This query can tell you if hardware acceleration is enabled, 
I don't see how this is relates to nested virtualization.
Am I missing something ?